src/chkstat.cpp

```diff
@@ -322,9 +322,13 @@ int Chkstat::processEntries() {
     size_t errors = 0;
 
     if (m_apply_changes && !m_have_proc) {
-        std::cerr << "ERROR: /proc is not available - unable to fix policy violations. Will continue in warn-only mode." << std::endl;
-        errors++;
-        m_apply_changes = false;
+        if (secure_getenv("CHKSTAT_FORCE_INSECURE_MODE_IF_NO_PROC") != nullptr) {
```

Comment:
I'm not sure about the name, `FORCE` sounds strange, like it *wants* to be insecure even if it might be possible otherwise. How about `ALLOW` instead?

Reply:
Once we have a final name, the environment variable should also be documented in the man page.